laravelServer/server/resources/views/welcome.blade.php

```diff
@@ -83,10 +83,10 @@
                                         {{ csrf_field() }}
                                         <input type="checkbox" name="status" class="onoffswitch-checkbox" id="placeholder3" tabindex="0" onchange="this.form.submit()" {{ $statuses->where('type', 3)->where('on', true)->isNotEmpty() ? 'checked' : '' }}>
                                         <label class="onoffswitch-label" for="placeholder3"></label>
-                                        <input type="hidden" name="type" value="3">
+                                        <input type="hidden" name="type" value="koelkast">
```

Comment:
currently type is a smallInt type of pgsql. So if we try to upload this our database will say wtf do you mean VARCHAR. wtf even is that. I want an int between -32,768 and 32,767